coverage.sh

```diff
@@ -10,7 +10,7 @@ npx buidler compile --config ./buidlerCoverageSol4.js
 
 if [ -n "$FILE" ]
 then
-    npx buidler coverage --config ./buidlerConfigSol5.js --testfiles $FILE
+    npx buidler coverage --config ./buidlerConfigSol5.js --solcoverjs ".solcover.js" --testfiles $FILE
 else
-    npx buidler coverage --config ./buidlerConfigSol5.js --testfiles "" --solcoverjs ".solcover.js" --temp ""
+    npx buidler coverage --config ./buidlerConfigSol5.js --solcoverjs ".solcover.js" --temp ""  --testfiles ""
```

Comment:
just changing order?
what is motivation for this?

Reply:
It is failed at Katalyst branch when you run
```
./coverage.sh -f test/sol5/kyberNetwork.js
```
This is a fix for coverage with single file